src/uct/api/v2/uct_v2.h

```diff
@@ -134,6 +206,19 @@ typedef struct {
 ucs_status_t
 uct_iface_estimate_perf(uct_iface_h tl_iface, uct_perf_attr_t *perf_attr);
 
+
+/**
+ * @ingroup UCT_MD
+ * @brief Undo the operation of @ref uct_md_mem_reg() and invalidate memory
+ *        region.
+ *
```

Comment:
Please add full documentation in addition to "brief" . Please add how v1 and different from v2